<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreTimeStepEditorWidget</class>
 <widget class="QWidget" name="CoreTimeStepEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="unitsTab">
      <attribute name="title">
       <string>Units</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="3" column="0">
        <widget class="QLabel" name="minutesLabel">
         <property name="text">
          <string>minutes:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="secondsLabel">
         <property name="text">
          <string>seconds:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="microsecondsLabel">
         <property name="text">
          <string>microseconds:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="milisecondsLabel">
         <property name="text">
          <string>miliseconds:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="hoursLabel">
         <property name="text">
          <string>hours:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="microsecondsSpinBox">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="milisecondsSpinBox">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="secondsSpinBox">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="minutesSpinBox">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="hoursSpinBox">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="predefinedTab">
      <attribute name="title">
       <string>Predefined</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="predefinedComboBox">
           <property name="currentText">
            <string>Custom</string>
           </property>
           <item>
            <property name="text">
             <string>Custom</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>94</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="formatLabel">
         <property name="text">
          <string>Format:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="formatComboBox">
         <item>
          <property name="text">
           <string>As time</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>As microseconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>As miliseconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>As seconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>As minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>As hours</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="timeStepValueLabel">
         <property name="text">
          <string>Time step value:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="timeStepLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>predefinedComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CoreTimeStepEditorWidget</receiver>
   <slot>predefinedChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hoursSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CoreTimeStepEditorWidget</receiver>
   <slot>stepChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>microsecondsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CoreTimeStepEditorWidget</receiver>
   <slot>stepChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>milisecondsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CoreTimeStepEditorWidget</receiver>
   <slot>stepChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>secondsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CoreTimeStepEditorWidget</receiver>
   <slot>stepChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minutesSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CoreTimeStepEditorWidget</receiver>
   <slot>stepChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>formatComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CoreTimeStepEditorWidget</receiver>
   <slot>formatChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>stepChanged(quint64)</signal>
  <slot>setStep(quint64)</slot>
  <slot>formatChanged(int)</slot>
  <slot>stepChanged()</slot>
  <slot>predefinedChanged(int)</slot>
 </slots>
</ui>
